diff --git a/cardano-crypto-praos/cardano-crypto-praos.cabal b/cardano-crypto-praos/cardano-crypto-praos.cabal
index 3f5da2e..eead1f0 100644
--- a/cardano-crypto-praos/cardano-crypto-praos.cabal
+++ b/cardano-crypto-praos/cardano-crypto-praos.cabal
@@ -42,7 +42,7 @@ flag external-libsodium-vrf
     description: Rely on a special libsodium fork containing the VRF code.
                  Otherwise expect a normal unaltered system libsodium, and
                  bundle the VRF code.
-    default: True
+    default: False
     manual: True
 
 library

